src/main/scala/com/horizon/exchangeapi/auth/IbmCloudModule.scala

```diff
@@ -397,12 +399,12 @@ object IbmCloudAuth {
     // This can throw exceptions OrgNotFound or IncorrectOrgFound
     val userQuery =
       for {
-        //associatedOrgId <- fetchOrg(userInfo) // can no longer use this, because the account id it uses to find the org is not necessarily unique...
-        //orgId <- verifyOrg(authInfo, userInfo, associatedOrgId)
-        orgAcctId <- fetchOrg(authInfo.org)
-        orgId <- verifyOrg(authInfo, userInfo, orgAcctId.flatten) // verify the org exists in the db, and in the public cloud case the cloud acct id of the apikey and the org entry match
+        //orgAcctId <- fetchOrg(authInfo.org)
+        //orgId <- verifyOrg(authInfo, userInfo, orgAcctId) // verify the org exists in the db, and in the public cloud case the cloud acct id of the apikey and the org entry match
+        orgId <- fetchVerifyOrg(authInfo, userInfo) // verify the org exists in the db, and in the public cloud case the cloud acct id of the apikey and the org entry match
         userRow <- fetchUser(orgId, userInfo)
         userAction <- {
+          logger.debug(s"userRow: $userRow")
```

Comment:
Can we actually leave this log in? Is it ok because it's a debug log?

Reply:
if you are worried about the pw, it is bcrypted